### ISSUE TITLE
how to make roulette to stop in the center of the prizeItem?

### ISSUE THREAD

**Issue by ngdevc** [IvanAdmaers/react-roulette-pro#7]
![image](https://user-images.githubusercontent.com/65579678/174246779-5e16b2de-31f2-48d4-98b3-76581df693a1.png)

**Comment by IvanAdmaers** [comment]
Hello, @tortuc. I implemented this feature in the package verison 2.1.2. Don't forget to update it. You should add this prop:
```jsx
<RoulettePro
  ...
  options={{ stopInCenter: true }}
  ...
/>
```

**Comment by ngdevc** [comment]
Thank you @IvanAdmaers 
It's been solved.
